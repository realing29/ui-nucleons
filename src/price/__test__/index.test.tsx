import React from 'react';
import { Price } from '..';
import { mount } from 'enzyme';

describe('Price', () => {
  it('should renders correctly', () => {
    const wrapper = mount(<Price value='123.45' currencyGrapheme='â‚½' />);

    expect(wrapper).toMatchSnapshot();
  });

  it('should handle props', () => {
    const wrapper = mount(
      <Price
        value='1234567.89'
        currencyGrapheme='$'
        crossedOut
        graphemeBefore
        className='test-class'
      />,
    );

    expect(wrapper).toMatchSnapshot();
  });
});
