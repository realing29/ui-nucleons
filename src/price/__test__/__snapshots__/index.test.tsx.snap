// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Price should handle props 1`] = `
<Price
  className="test-class"
  crossedOut={true}
  currencyGrapheme="$"
  graphemeBefore={true}
  value="1234567.89"
>
  <span
    className="root test-class crossed-out"
  >
    $ 1 234 567,89
  </span>
</Price>
`;

exports[`Price should renders correctly 1`] = `
<Price
  currencyGrapheme="₽"
  value="123.45"
>
  <span
    className="root"
  >
    123,45 ₽
  </span>
</Price>
`;
